src/main/java/org/mskcc/cbio/oncokb/web/rest/UsageAnalysisController.java

```diff
@@ -0,0 +1,144 @@
+package org.mskcc.cbio.oncokb.web.rest;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import javax.annotation.PostConstruct;
+
+import com.amazonaws.auth.AWSCredentials;
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.S3Object;
+import com.amazonaws.services.s3.model.S3ObjectInputStream;
+import com.google.gson.Gson;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.mskcc.cbio.oncokb.config.application.ApplicationProperties;
+import org.mskcc.cbio.oncokb.web.rest.vm.usageAnalysis.UsageSummary;
+import org.mskcc.cbio.oncokb.web.rest.vm.usageAnalysis.UserOverviewUsage;
+import org.mskcc.cbio.oncokb.web.rest.vm.usageAnalysis.UserUsage;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/api")
+public class UsageAnalysisController {
+
+    @Autowired
+    private ApplicationProperties applicationProperties;
+
+    private String s3AccessKey = "";
+    private String s3SecretKey = "";
+    private String s3Region = "";
+
+    @PostConstruct
+    private void init(){
+        s3AccessKey = applicationProperties.getAws().getS3AccessKey();
+        s3SecretKey = applicationProperties.getAws().getS3SecretKey();
+        s3Region = applicationProperties.getAws().getS3Region();
+    }
+
+    @GetMapping("/usage/user/{userId}")
+    public ResponseEntity<UserUsage> userUsageGet(@PathVariable String userId)
+        throws IOException, ParseException {
+    
+        HttpStatus status = HttpStatus.OK;
+
+        AWSCredentials credentials = new BasicAWSCredentials(s3AccessKey, s3SecretKey);
+        AmazonS3 s3client = AmazonS3ClientBuilder.standard()
+                .withCredentials(new AWSStaticCredentialsProvider(credentials)).withRegion(s3Region).build();
+
+        S3Object s3object = s3client.getObject("oncokb", "usage-analysis/userSummary.json");
+        S3ObjectInputStream inputStream = s3object.getObjectContent();
+        JSONParser jsonParser = new JSONParser();
+        JSONObject jsonObject = (JSONObject) jsonParser.parse(new InputStreamReader(inputStream, "UTF-8"));
```

Comment:
Since we are getting the content all from the same bucket, let's wrap them to a function and return the JSON. Also put the file names to a constant